### PR TITLE
mesa: update to 24.1.3.

### PR DESCRIPTION
**Pull request by SpidFightFR**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x64-GlibC)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x64-musl

### CODE REVIEW

**Comment by chrysos349** [comment]
#51076

**Comment by SpidFightFR** [comment]
> #51076

noted those, the changes in this pr don't conflict.

**Comment by Calandracas606** [comment]
I've updated my PR to also bump mesa to 24.1.3

File: srcpkgs/mesa/template [hunk 1]
```diff
@@ -23,8 +23,8 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
 changelog="https://docs.mesa3d.org/relnotes.html"
-distfiles="https://mesa.freedesktop.org/archive/mesa-${version}.tar.xz"
-checksum=a2c584c8d57d3bd8ba11790a6e9ae3713f8821df96c059b78afb29dd975c9f45
+distfiles="https://archive.mesa3d.org/mesa-${version}.tar.xz"
+checksum=63236426b25a745ba6aa2d6daf8cd769d5ea01887b0745ab7124d2ef33a9020d
```
Comment (Calandracas606) [inline srcpkgs/mesa/template:27]: why is the distfiles url changed?

Comment (SpidFightFR) [reply srcpkgs/mesa/template:27]: > why is the distfiles url changed?

They changed the url, if you use the base url, you'll be redirected on the following new site.

**Comment by Calandracas606** [comment]
i don't know if this should be closed.

its possible that the maintainers would want to update mesa, but not use llvm18 yet

**Comment by SpidFightFR** [comment]
> i don't know if this should be closed.
> 
> its possible that the maintainers would want to update mesa, but not use llvm18 yet

Alright. It's reopened, i subbed to your PR so that i can keep track of the decision maintainers went with. 😄👍

**Comment by Calandracas606** [comment]
i rebased https://github.com/void-linux/void-packages/pull/51076 this morning, and CI passes without issue

**Comment by dkwo** [comment]
my bad, please disregard

**Comment by SpidFightFR** [comment]
> my bad, please disregard

ight, sorry for the late response, imma do it anyway just in case.

**Comment by SpidFightFR** [comment]
Can confirm, it works.

**Pull request closed without merging**